CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+## 2.5.0-rc.0 / 2018-10-23
+
+Starting with the s390x platform for Linux is officially supported.
```

Comment:
* "Starting with this release"?

Reply:
We tend to not list changes like build stuff in our release notes.

Reply:
@SuperQ agreed, but supporting an additional platform seems like it belongs in the changelog. I'm actually interested what we mean by this (I took the "support" statement right from the PR that merged this). For this and other platforms that are not amd64 we should probably clearer on what support means as we only run tests on amd64 and arguably all of us probably don't run Prometheus on anything other (maybe some arm devices). So "supportability" becomes somewhat undefined.